### PR TITLE
workflow

### PR DESCRIPTION
**Pull request by sebwalle**
Added a unit test check

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Go Unit Tests
+
+on:
+  push:
```
Comment (furst) [inline .github/workflows/test.yml:4]: Do we need to run this when we merge to master? I think it should be enough to only run on PR

Comment (sebwalle) [reply .github/workflows/test.yml:4]: Not sure if we have push to master enabled or not

Comment (furst) [reply .github/workflows/test.yml:4]: I'll add a branch block as soon as this is merged! There is also a setting to require latest code before merge. Thing is with running this on merge to master is no one will see it

Comment (sebwalle) [reply .github/workflows/test.yml]: okok, changed it now

**Review by furst — APPROVED** [review]

**Pull request merged**